### PR TITLE
Add temporarily cuts for Germany in 2020

### PR DESCRIPTION
**Pull request by mupat**
In order to support the temporarily cut offs for vat in germany in 2020, I've added them to the json and added some tests to make sure they work.

### CODE REVIEW

File: tests/mocha/tests.js [hunk 1]
```diff
@@ -198,4 +198,19 @@ describe('VATCountry()', function() {
       expect(vatRates.getCountry('DK').getParkingRate()).to.be.a('undefined');
     });
   });
+
+  describe('limitedCutsForGermany', function() {
+    it('should return 16% and 5% after 2020-07-01', function() {
+      vatRates.setDate('2020-07-01');
+      debugger
```
Comment (Betree) [inline tests/mocha/tests.js]: We should probably remove that

```suggestion
```

**Review by Betree — COMMENTED** [review]
That looks great, thanks for taking the initiative @mupat :raised_hands:

File: tests/mocha/tests.js (hunk 1 above)
Comment (mupat) [reply tests/mocha/tests.js]: thx, done

**Comment by staaky** [comment]
Thanks! I've merged it into 2.0.3.

**Pull request merged**